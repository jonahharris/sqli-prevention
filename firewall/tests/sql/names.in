>>> Sada testovacich SQL dotazu: identifikatory
# Obecne identifikatory
SELECT * FROM abc;
SELECT * FROM 1abc;
SELECT * FROM $abc;
SELECT * FROM ab$cd;
SELECT * FROM ab_cd;
SELECT * FROM `123%`;
SELECT * FROM `TABLE`;
SELECT * FROM `čeština`;
SELECT * FROM t2.`čeština`;
SELECT * FROM `abc"d`;
CREATE TABLE `*` (a INT);
SELECT t1.i FROM t1;
SELECT test.t1.i FROM t1;
SELECT test . t1 . i FROM t1;
SELECT t1.i FROM test.t1;
# Uzivatelske promenne
SELECT @x IS NULL;
SELECT @`var_name` IS NULL;
SELECT @'var_name' IS NULL;
SELECT @"var_name" IS NULL;
SET @a = 0;
SET @a = 'SELECT * FROM A';
SET @a.b = 42;
SET @a$b = 444;
SET @a_b = 111;
SET @a = 1, @b = 2, @c = 3;
SELECT @a, @b := @a + @b;
