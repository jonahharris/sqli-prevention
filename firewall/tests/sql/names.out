>>> Sada testovacich SQL dotazu: identifikatory
# Obecne identifikatory
SELECTALL;TABLE abc;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE 1abc;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE $abc;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE ab$cd;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE ab_cd;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE 123%;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE TABLE;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE čeština;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE t2.čeština;SELECT 0 1 1;STMT;1;0;
SELECTALL;TABLE abc"d;SELECT 0 1 1;STMT;1;0;
TABLE *;COLUMN 40000 a;CREATE 0 0 0 1;STMT;1;0;
FIELDNAME t1.i;TABLE t1;SELECT 0 1 1;STMT;1;0;
FIELDNAME test.t1.i;TABLE t1;SELECT 0 1 1;STMT;1;0;
FIELDNAME test.t1.i;TABLE t1;SELECT 0 1 1;STMT;1;0;
FIELDNAME t1.i;TABLE test.t1;SELECT 0 1 1;STMT;1;0;
# Uzivatelske promenne
USERVAR x;ISNULL;SELECTNODATA 0 1;STMT;1;0;
USERVAR `var_name`;ISNULL;SELECTNODATA 0 1;STMT;1;0;
USERVAR 'var_name';ISNULL;SELECTNODATA 0 1;STMT;1;0;
USERVAR "var_name";ISNULL;SELECTNODATA 0 1;STMT;1;0;
NUMBER 0;SET a;STMT;1;0;
STRING '***';SET a;STMT;1;0;
NUMBER 42;SET a.b;STMT;1;0;
NUMBER 444;SET a$b;STMT;1;0;
NUMBER 111;SET a_b;STMT;1;0;
NUMBER 1;SET a;NUMBER 2;SET b;NUMBER 3;SET c;STMT;1;0;
USERVAR a;USERVAR a;USERVAR b;ADD;ASSIGN @b;SELECTNODATA 0 2;STMT;1;0;
